web/src/routes/Export.jsx

```diff
@@ -22,6 +27,10 @@ export default function Export() {
   const [endDate, setEndDate] = useState(localISODate);
   const [endTime, setEndTime] = useState('23:59');
 
+  useEffect(() => {
```

Comment:
Why not use swr here?

Reply:
Is there a way to make swr not use the configured base path? The default base path includes /api

Reply:
Actually I think I see what you mean, I'll fix that